popup.html

```diff
@@ -10,6 +10,7 @@
   font-family: "Segoe UI", Arial, Meiryo, sans-serif;
   max-width: 600px;
   max-height: 600px;
+  min-height: 480px;
```

Comment:
@fleaz Is this definition essential? Or does it need to be 480px?

Reply:
I just added this because otherwise the window will be very small when you have a little number of accounts configured, which doesn't looked  good to me. But the actual value is not important

Reply:
Can you provide a screenshot of how small it is? I think that the height of the menu on the right side is sufficient. If it's a 2k monitor, this value is too large, on the contrary.

Reply:
I gave it another look and the height is indeed sufficient. Also most of the user of this plugin will have a load of accounts so this should be a non issue.